debian/control

```diff
@@ -9,6 +9,7 @@ Build-Depends: debhelper (>= 8),
                python3-all,
                python3-setuptools,
                python-nose, python3-nose, python-mock,
+               python-enum34 python3-enum34,
```

Comment:
:+1: cool. didn't know this was already in trusty too!

do we need a python3 version since it's in the stdlib?